query/src/main/scala/filodb/query/exec/PromQlRemoteExec.scala

```diff
@@ -89,16 +92,24 @@ case class PromQlRemoteExec(queryEndpoint: String,
                             params: PromQlQueryParams,
                             numberColumnRequired: Boolean = false) extends RemoteExec {
 
-  private val columns = Seq(ColumnInfo("timestamp", ColumnType.TimestampColumn),
+  override def args: String = s"${params.toString}, queryEndpoint=$queryEndpoint, " +
+    s"requestTimeoutMs=$requestTimeoutMs, limit=${queryContext.sampleLimit}, " +
+    s"numberColumnRequired=${numberColumnRequired}"
+  private val defaultColumns = Seq(ColumnInfo("timestamp", ColumnType.TimestampColumn),
     ColumnInfo(if (numberColumnRequired) "number" else "value", ColumnType.DoubleColumn))
-  private val recSchema = SerializedRangeVector.toSchema(columns)
-  private val resultSchema = ResultSchema(columns, 1)
+  private val defaultRecSchema = SerializedRangeVector.toSchema(defaultColumns)
+  private val defaultResultSchema = ResultSchema(defaultColumns, 1)
+
+  val histColumns = Seq(ColumnInfo("timestamp", ColumnType.TimestampColumn),
+    ColumnInfo("h", ColumnType.HistogramColumn))
+  val histRecSchema = SerializedRangeVector.toSchema(histColumns)
+  val histResultSchema = ResultSchema(histColumns, 1)
```

Comment:
Rest of the variables are `private`. So does it make sense to keep `histColumns`, `histRecSchema`, `histResultSchema` as private too?